validator/publication.py

```diff
@@ -1,27 +1,19 @@
-import requests
 from validator.generic import GenericValidator
+from validator.request.connector import Connector, NotFound
```

Comment:
I might have missed something but shouldn't it be `DefaultConnector` there instead of the abstract `Connector` ?

Reply:
Same here, it's abstracted